add-column.md

```diff
@@ -18,6 +18,104 @@ The user must have the `CREATE` [privilege](privileges.html) on the table.
 
 ## Parameters
 
+| Parameter | Description |
+|-----------|-------------|
+| `table_name` | The name of the table to which you want to add the column. |
+| `name` | The name of the column you want to add. The name must be unique to the database and follow these [identifier rules](keywords-and-identifiers.html#identifiers).  |
```

Comment:
The second sentence isn't quite right. It's fine for multiple tables to have identically named columns (for example, many tables might have an `id` column). It's within a single table that you can't have 2 columns with the same name. So I'd suggest something like:

> The column name must follow these [identifier rules](keywords-and-identifiers.html#identifiers) and must be unique within the table but can have the same name as indexes or constraints.

Got that last bit from the `column_def` parameter in the `create-table.md` docs.